packages/@angular/cli/blueprints/ng2/files/e2e/tsconfig.json

```diff
@@ -11,7 +11,7 @@
     "moduleResolution": "node",
     "outDir": "../dist/out-tsc-e2e",
     "sourceMap": true,
-    "target": "es5",
+    "target": "es6",
```

Comment:
The commit message says `ES5 is necessary in projects because Uglify does not support ES6` but here you're setting `es6`. Did I misunderstand it?